tests/x11/smt.pm

```diff
@@ -21,8 +21,12 @@ use repo_tools;
 sub run {
     x11_start_program('xterm -geometry 150x35+5+5', target_match => 'xterm');
     become_root;
+    # run btrfs-balance to prevent high load in background
+    assert_script_run "/usr/share/btrfsmaintenance/btrfs-balance.sh";
+    # diable screensaver as well
+    assert_script_run "dbus-launch gsettings set org.gnome.desktop.session idle-delay 0";
```

Comment:
Do we need this changes now? I had all runs without these extra changes. So I would aim having those in force_cron_run replacement. As for screensaver, we should rather fix the call we have in disabling screensaver.